cli/Cargo.toml

```diff
@@ -14,7 +14,7 @@ cast = { path = "../cast" }
 evm-adapters = { path = "../evm-adapters" }
 
 # ethers = "0.5"
-ethers = { git = "https://github.com/gakonst/ethers-rs", default-features = false }
+ethers = { git = "https://github.com/pyk/ethers-rs", branch = "optimism-support", default-features = false }
```

Comment:
idt ppl are gonna agree to this heh, probably best to wait for https://github.com/gakonst/ethers-rs/pull/737 to be merged